actors/builtin/miner/monies.go

```diff
@@ -64,15 +73,18 @@ func PledgePenaltyForTermination(initialPledge abi.TokenAmount, sectorAge abi.Ch
 // newly-committed power, holding the per-epoch block reward constant (though in reality it will change over time).
 // The network total pledge and circulating supply parameters are currently unused, but may be included in a
 // future calculation.
-func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
+func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower, baselinePower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
 	// Details here are still subject to change.
 	// PARAM_FINISH
 	// https://github.com/filecoin-project/specs-actors/issues/468
-	_ = networkCirculatingSupply // TODO: ce use this
-	_ = networkTotalPledge       // TODO: ce use this
+	_ = networkTotalPledge // TODO: ce use this
 
-	if networkQAPower.IsZero() {
-		return epochTargetReward
-	}
-	return big.Mul(InitialPledgeFactor, ExpectedDayRewardForPower(epochTargetReward, networkQAPower, qaPower))
+	ipBase := big.Mul(InitialPledgeFactor, ExpectedDayRewardForPower(epochTargetReward, networkQAPower, qaPower))
+	additionalIPNum := big.Mul(big.Mul(LockTargetFactorNum, networkCirculatingSupply), qaPower)
+	storageToCover := big.Max(networkQAPower, baselinePower)
```

Comment:
How about defining `pledgeShareNum := qaPower` and renaming `storageToCover` to `pledgeShareDenom`? That would map this a lot closer to how cryptoecon defines it. It would look like this:

``` golang
lockTargetNum := big.Mul(LockTargetFactorNum, networkCirculatingSupply)
lockTargetDenom := LockTargetFactorDenom
pledgeShareNum := qaPower
pledgeShareDenom := big.Max(big.Max(qaPower, storageToCover), qaPower)
additionalIPNum := big.Mul(lockTargetNum, pledgeShareNum)
additionalIPDenom := big.Mul(lockTargetDenom, pledgeShareDenom)
additionalIP := big.Div(additionalIPNum, additionalIPDenom)
```

A little more verbose, but maps cleanly to the math and has the same numerical properties.

Reply:
You could write a little helper struct to clean this up this rational math on big integers a bit, but if you do, you have to promise you'll abbreviate its name to `BigRat`.

Reply:
![image](https://user-images.githubusercontent.com/5515260/87818542-bba38100-c838-11ea-987f-9d285e3a89ad.png)

Reply:
I like the idea but will file an issue for doing using the big rat consistently

Reply:
Feel free to add more details on #700